src/types/sql/db.ts

```diff
@@ -29,6 +30,7 @@ export interface Database {
   otps: SqlOTP;
   passwords: SqlPassword;
   sessions: SqlSession;
+  templates: SqlTemplate;
```

Comment:
what are these? we have templates in strings but also in SQL?

Reply:
Auth0 exposes templates via the api so was thinking about moving it there, but... then I changed my mind :) I'll remove these.